src/io/openshift/plugins/analytics.groovy

```diff
@@ -1,10 +1,28 @@
 package io.openshift.plugins
 
 import io.openshift.Events
+import io.openshift.Utils
 
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime()
+        if(!image) {
+            return
+        }
+        def url = Utils.shWithOutput(this, "git config remote.origin.url")
+        try {
+            retry(3) {
+                def response = bayesianAnalysis url: 'https://bayesian-link', gitUrl: url, ecosystem: image
```

Comment:
@yzainee could you please substitute `gitUrl: url` with `gitUrl: e.git.url`. It already coming in the `event argument`.

Reply:
`event argument` also has `git.commit` which is a commit id used for the build